modules/notification/webhook/webhook.go

```diff
@@ -10,7 +10,6 @@ import (
 	"code.gitea.io/gitea/modules/notification/base"
 	"code.gitea.io/gitea/modules/setting"
 	api "code.gitea.io/gitea/modules/structs"
-	"code.gitea.io/gitea/modules/webhook"
```

Comment:
@lunny it works on your current branch but not avter merge!
-> we still need this import